test/test_scanner.py

```diff
@@ -988,6 +988,45 @@ def _file_test(self, url, filename, package, version):
                     "CVE-2017-17522"
                 ],
             ),
+            (
+                "test-py_thon-3.7.1_1.out",
```

Comment:
Do we still need the `py_thon` filename if we're taking out the python filename check already?